src/open_inwoner/scss/admin/_admin_theme.scss

```diff
@@ -101,9 +100,13 @@ div.breadcrumbs {
 #content {
   /* adjusting for django-cms page */
   #changelist:not([class~='cms-pagetree-root']) {
-    display: grid;
-    grid-template-columns: 1fr 360px;
+    display: block;
```

Comment:
I'm not sure what causes this (or if it is an issue at all, because I know very little about django-cms), but if I log in as an admin, I don't get to see the CMS toolbar if I navigate to the frontend (if I try the same on develop it does show the toolbar):

![image](https://github.com/maykinmedia/open-inwoner/assets/29249171/b919819b-041c-492c-a91b-b37b3101e034)

I can only get the toolbar to show if I go to a page in the admin and then click the view button
![Screenshot 2024-02-19 at 14-47-10 Open Inwoner Deventer](https://github.com/maykinmedia/open-inwoner/assets/29249171/98689cd6-b297-4bd0-b452-aa7a7f8cf5ff)

Reply:
This might be related to the changes to `DropToolbarMiddleware`. I'll look into it.